.azure-pipelines/callGithubWorkflow.yaml

```diff
@@ -60,7 +62,40 @@ jobs:
         # NOT MAIN/MASTER OR AUTOMATED BRANCH
         if ($branchName -ne 'master' -and $isAutoGeneratedPR -eq $false -and $targetBranch -eq 'master')
         {
-          .script/package-automation/invokeGithubWorklow.ps1 $branchName $pullRequestNumber
+          # INVOKE GITHUB WORKFLOW
+          $header = @{
+              "Accept" = "application/vnd.github+json"
+          }
+
+          $githubToken = "$($env:GITHUB_TOKEN_VALUE)"
+          $token = $githubToken | ConvertTo-SecureString -AsPlainText -Force
+          $pullRequestNumberInt = [int]$pullRequestNumber
+          $client_payload = @{
+              "command" = "ping"
+              "pullRequestBranchName" = "$branchName"
+              "pullRequestNumber" = $pullRequestNumberInt
```

Comment:
Is there a need to pass the PR number?

Reply:
yes we need it in our github workflow run